core/chainlink.goreleaser.Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Final image: ubuntu with chainlink binary
+FROM ubuntu:20.04
+
+ARG CHAINLINK_USER=root
+ARG TARGETARCH
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get update && apt-get install -y ca-certificates gnupg lsb-release curl
+
+# Install Postgres for CLI tools, needed specifically for DB backups
+RUN curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add - \
+  && curl https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - \
+  && echo "deb http://apt.postgresql.org/pub/repos/apt/ `lsb_release -cs`-pgdg main" |tee /etc/apt/sources.list.d/pgdg.list \
+  && apt-get update && apt-get install -y postgresql-client-14 \
+  && apt-get clean all
+
+COPY . /usr/local/bin/
+COPY ./tmp/linux_${TARGETARCH}/libs /usr/local/bin/libs
+
+RUN if [ ${CHAINLINK_USER} != root ]; then \
```

Comment:
I would check with @chainchad what our deprecation cycle is for root. Maybe we can nuke this now

Reply:
We'll need to support the root images for a little while longer, unfortunately.